src/parachain/transaction.ts

```diff
@@ -181,6 +181,35 @@ export class DefaultTransactionAPI implements TransactionAPI {
         }
     }
 
+    static async waitForEvent<T extends AnyTuple>(
```

Comment:
Remove, not used

Reply:
It's used in the bot, but I can move it there to avoid it being removed from the lib again